### PR TITLE
DEV-724: use stdout by default for cictl

### PR DESCRIPTION
**Pull request by aelkiss**

### CODE REVIEW

**Review by moseshll — APPROVED** [review]
This does the right thing. I wonder, however, how this might translate into a modified command for the Argo workflow. I guess just using "daily" in the Argo command would suffice to get us a datestamped file. It's irritating that those are the only built-ins for requesting a log named by a format we're accustomed to. Maybe it's overkill now that we will likely no longer use anything other than `cictl index continue` except in oddball circumstances, but a standard file name with the name of the cictl subcommand and a datestamp would likely have been the way to go.

**Comment by aelkiss** [comment]
Let me see what I can do...

File: spec/cictl/.index_command_spec.rb.swp
Comment (moseshll) [inline spec/cictl/.index_command_spec.rb.swp:1]: Inadvertent commit, I assume? Could .gitignore.

**Review by moseshll — APPROVED** [review]
This looks good. Odd `.swp` to be removed, otherwise APPROVE.

I look forward to playing with `timecop`.

**Pull request merged**